src/TLuaInterpreter.cpp

```diff
@@ -13769,3 +13770,40 @@ int TLuaInterpreter::startPermSubstringTrigger(const QString & name, const QStri
 
 }
 
+int TLuaInterpreter::alert( lua_State * L )
+{
+    int luaAlertDuration;
+
+    int n = lua_gettop( L );
+    if( n == 0 )
+    {
+        // msec argument for QApplication::alert() is optional (defaults to 0)
+        luaAlertDuration = 0;
+    }
+    else
+    {
+        if( ! lua_isnumber( L, 1 ) )
+        {
+            lua_pushfstring( L, "alert: bad argument #1 type (alert duration in seconds as number expected, got %s!)",
+                            luaL_typename( L, 1 ));
+            lua_error( L );
+            return 1;
+        }
+        else
+        {
+            luaAlertDuration = lua_tointeger( L, 1 );
+
+            if ( luaAlertDuration < 0 )
```

Comment:
...with a `double` one has to be a bit circumspect about testing for precise values but I think a `< 0.000` would still work here in that case...